internal/svc/list.go

```diff
@@ -165,3 +166,17 @@ func getPolicyVersion(p *policyv1.Policy) string {
 	}
 }
 
+func sortSchemas(schemas []*schemav1.SchemaWithoutDef) {
+	sort.Sort(nameOrderSchema(schemas))
+}
+
+type nameOrderSchema []*schemav1.SchemaWithoutDef
+
+func (s nameOrderSchema) Len() int { return len(s) }
+
+func (s nameOrderSchema) Swap(i, j int) { s[i], s[j] = s[j], s[i] }
+
+func (s nameOrderSchema) Less(i, j int) bool {
+	in, jn := s[i].Id, s[j].Id
+	return in < jn
+}
```

Comment:
This can be simpler:

```suggestion
func sortSchemas(schemas []*schemav1.SchemaWithoutDef) {
	sort.Slice(schemas, func(i,j int) bool{
	    return schemas[i].id < schemas[j].id
	})
}
```